From 911ceb8d030151de5f4c2296964c9acebf321f21 Mon Sep 17 00:00:00 2001
From: Jianming Qiao <jianming.qiao@bp.renesas.com>
Date: Sat, 14 Sep 2019 18:12:48 +0100
Subject: [PATCH] Fix asm mov error

The asm code is incorrect and the following message will appear:
"Error: immediate expression requires a # prefix -- `mov r0,0'"

Add # will fix the issue.

Signed-off-by: Jianming Qiao <jianming.qiao@bp.renesas.com>
---
 tensorflow/lite/experimental/ruy/pack_arm.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tensorflow/lite/experimental/ruy/pack_arm.cc b/tensorflow/lite/experimental/ruy/pack_arm.cc
index 7e0814546e12..4f8f9ac1f813 100644
--- a/tensorflow/lite/experimental/ruy/pack_arm.cc
+++ b/tensorflow/lite/experimental/ruy/pack_arm.cc
@@ -1289,7 +1289,7 @@ void PackFloatNeonOutOfOrder(const float* src_ptr0, const float* src_ptr1,
 
           "ands r2, %[rows], #3\n"
           "beq 4f\n"
-          "mov r0, 0\n"
+          "mov r0, #0\n"
           // Zero out q0 - q3
           "vdup.32 q0, r0\n"
           "vdup.32 q1, r0\n"
-- 
2.17.1

